### PR TITLE
[docs] update package, features and config mode docs

### PR DESCRIPTION
**Pull request by maurerle**
This is a series of docs improvements created from #3131 

I tested the docs using `make -C docs html && python -m http.server --directory docs/_build/html`.

I am open for additions and suggestions.
If I should split the commits into separate PRs I can sure do that too, though I did not want to create 5 PRs about docs.

I think that especially some visualizations to refer to are quiet important for users too, so I added two pictures.

### CODE REVIEW

**Comment by maurerle** [comment]
@skorpy2009 would you like to have a look at this, please?

**Comment by blocktrron** [comment]
Pleases prefix and separate the commits by topic.

File: docs/features/tls.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+TLS support
+===========
+
+The generic TLS implementation which is currently used by OpenWRT can be installed or added as dependency through the package ``gluon-tls``.
+This removes the need for community packages to depend on a specific TLS implementation (like mbedtls, OpenSSL or WolfSSL).
+
+This package is a placeholder for the current TLS implementation used.
```
Comment (herbetom) [inline docs/features/tls.rst:7]:
```suggestion
This package is an alias for the current TLS implementation used.
```

File: docs/package/gluon-web-cellular.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.. _package-gluon-web-cellular:
+
+gluon-web-cellular
+==================
+
+This package allows the configure WWAN for capable cellular devices.
```
Comment (herbetom) [inline docs/package/gluon-web-cellular.rst:6]:
```suggestion
This package allows to configure WWAN for capable cellular devices.
```

File: docs/package/gluon-web-cellular.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+This works by creating an abstraction layer into Gluon which takes common options (SIM PIN / APN) and translates it to modem-specific settings based on the specific device using. 
+Doing so limits the use-case onto specific models (no LTE sticks possible) but provides a common interface.
+
+The WWAN is assigned the firewall WAN zone and wired WAN can still be used, however without prioritization.
```
Comment (herbetom) [inline docs/package/gluon-web-cellular.rst:11]:
```suggestion
The WWAN is assigned the WAN firewall zone and wired WAN can still be used, however without prioritization.
```

File: docs/package/gluon-web-network.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
+gluon-web-network
+=================
+
+The package *gluon-web-network* is part of the :ref:`Feature Flag <user-site-feature-flags>` web-advanced.
```
Comment (herbetom) [inline docs/package/gluon-web-network.rst:4]:
```suggestion
The package *gluon-web-network* is part of :ref:`Feature Flag <user-site-feature-flags>` web-advanced.
```

File: docs/package/gluon-web-network.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+It allows to configure the roles of the gluon nodes network interfaces in config mode through checkboxes.
```
Comment (herbetom) [inline docs/package/gluon-web-network.rst:5]:
```suggestion
It allows to configure the network interfaces roles of the gluon node in config mode through checkboxes.
```

**Review by herbetom — COMMENTED** [review]
not a complete review, just what i noticed at a glance

File: docs/package/gluon-web-network.rst (hunk 4 above)
Comment (maurerle) [reply docs/package/gluon-web-network.rst:4]: I think this is correct before in the way that you say:
The scene is part of the Movie XYZ
and not
The scene is part of Movie XYZ?

Though one probably can say both..?

Comment (grische) [reply docs/package/gluon-web-network.rst:4]: @maurerle 
image

**Comment by maurerle** [comment]
I would like to get this merged :)
Is there anything else I should do or improve?

**Comment by maurerle** [comment]
I update the PR to reflect to latest changes of the web-cellular package in #3307 
See: https://github.com/freifunk-gluon/gluon/compare/68e431003d183ed474d9b552482f6aeeb0cb9a3c..ecc12e46d4e950c3f30e2dc8b7f9c97000aaa050

The build of the resulting docs can be found here:
https://gluon--3188.org.readthedocs.build/en/3188/ ("Details" link from the CI runs)

**Review by rotanid — CHANGES_REQUESTED** [review]
i'd merge it, if you revert your wlan/wifi change.
wifi is a trademark and we want to use wlan.
if there is something like "wifi", please change that one in turn to "wlan", but not the other way around.

**Comment by maurerle** [comment]
Thanks @rotanid - I removed this commit through a rebase.
Lets merge?

**Review by rotanid — APPROVED** [review]

**Pull request merged**